vmd/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Vmd(AutotoolsPackage):
```

Comment:
It looks to me like this package doesn't use Autotools, it just happens to contain a `configure` script. I would inherit the `Package` base class, add:
```python
phases = ['configure', 'build', 'install']
```
then fill out those methods.